cardinal/lazycontainer.go

```diff
@@ -1,4 +1,4 @@
-package ecs
+package cardinal
```

Comment:
i think the lazy container type and constructor can both be made private? looks like this is only used internally